<h2>Term Test A Solutions</h2>

<h4>Question 1</h4>

2022: 3.2

A rocket, fired from rest at time $t=0$, has an initial
mass of $m_0$ (including its fuel). Assuming that the fuel is consumed
at a constant rate $k$, the mass $m$ of the rocket, while fuel is being
burned, will be given by $m(t)=m_{0}-kt$. It can be shown that if air resistance
is neglected and the fuel gases are expelled at a constant speed $c$
relative to the rocket, then the velocity $v$ of the rocket will satisfy
the equation
\begin{equation}
  \label{eq:fcgupphi}
  m\frac{dv}{dt}=ck-mg\notag
\end{equation}
$g=9.8\,N/s^2$ is the acceleration due to gravity.
* Find $v(t)$ keeping in mind
that the mass $m$ is a function of $t$.
* Suppose that the fuel accounts for 70\% of the initial mass of
  the rocket and that all of the fuel is consumed at $60\,s$. Find the
  velocity of the rocket in metres per second at the instant the fuel
  is exhausted. Take $c=2500\,m/s$ and $m_{0}=1.2\times{}10^{6}\,kg$.
  Round to nearest whole number.

In [2]:
g=9.8

9.8

Use separation for
$$
v(t)=-c\ln\left(m_0-kt\right)-gt+C
$$
with
$$
C=c\ln{}m_0
$$
Alternative:
$$
v(t)=-c\ln\left(\frac{m_0}{k}-t\right)-gt+D
$$

In [18]:
c=2500

2500

In [19]:
m0=1.2*10^6

1.2e6

In [20]:
k=(0.7*m0)/60

14000.0

In [21]:
using Roots

In [26]:
C=c*log(m0)

34994.58028689557

In [27]:
v1=(-c)*log(m0-k*60)-g*60+C

2421.932010814842

The rocket travels at approximately $2422\,m/s$ when the fuel is consumed.

<h4>Question 2</h4>

2022: 3.6

The family of solutions to the differential equation:
\begin{equation}
  \label{eq:hjyfdrvb}
  \frac{dy}{dx}=\frac{x^{2}-y^{2}}{2xy}\notag
\end{equation}
trace out the streamlines of a planar flow in which the fluid moves
according to the velocity vector:
\begin{equation}
  \label{eq:qldovisp}
  \vec{v}=v_{x}\vec{i}+v_{y}\vec{j}=\left(2xy\right)\vec{i}+\left(x^{2}-y^{2}\right)\vec{j}\notag
\end{equation}
Determine the general solution to the differential equation above. Do
not apply boundary conditions.

You can use homogeneous equations for
$$
x\frac{du}{dx}+u=\frac{1}{2u}-\frac{u}{2}
$$
with $ux=y$; or exact equations for
$$
(x^2-y^2)dx+(-2xy)dy=0
$$
or Bernoulli for
$$
\frac{dy}{dx}+\frac{1}{2x}y=\frac{x}{2}y^{-1}
$$
to yield
$$
y^{2}=\frac{x^{2}}{3}+\frac{C}{x}
$$

<h4>Question 3</h4>

2022: 2.5

The shape of a curved mirror that reflects light from a
source at the origin (the light bulb in the accompanying diagram) into
a beam of rays parallel to the $x$-axis is governed by the differential
equation:
\begin{equation}
  \label{eq:wggryuwc}
  \frac{d^2y}{dx^2}=-\frac{1}{y}\left(\frac{dy}{dx}\right)^2\notag
\end{equation}
Solve this differential equation to find the equation of the shape of
the curve. Use the fact that, in the illustrated coordinate system,
the mirror surface passes through the points $(-1,0)$ and $(0,2)$ to
determine the value of any arbitrary constants in the solution.

Use reduction of order: $p=\frac{dy}{dx}$ and $\frac{d^2y}{dx^2}=\frac{dp}{dy}p$ for
$$
\frac{dp}{dy}=-\frac{p}{y}
$$
which is separable with the result $Ayp=1$. Now
$$
x=\int\,dx=\int\frac{1}{p}\,dy=\int{}Ay\,dy=\frac{1}{2}Ay^2+B
$$
Clean this up for $y^2=A(x-B)$ and find $A=4,B=-1$ using the initial conditions. The parabola is
$$
y^{2}=4(x+1)
$$

<h4>Question 4</h4>

2022: 3.7

In the unlikely event that an elevator's cables are
severed, numerous safeties kick in. For a particular elevator of mass
$m$, the impact at the bottom of the elevator shaft is cushioned by
both a massive metal spring with spring constant $k$ and a hydraulic
shock absorber (a piston in an oil-filled cylinder) with damping
constant $c$. Safety clamps also grip steel rails running up the side
of the shaft. The vertical force exerted by these clamps is $F_{0}$ at
the moment ($t=0$) that the elevator car impacts the safety system at
the bottom of the shaft and thereafter falls off exponentially.

The motion after the moment of impact is governed by the differential
equation:
\begin{equation}
  \label{eq:ksjppqng}
  m\frac{d^2y}{dt^2}+c\frac{dy}{dt}+ky=-mg+F_{0}e^{-2.5t}\notag
\end{equation}
Use the method of undetermined coefficients to determine the full
general solution for $y$ as a function of time $t$ with
\begin{align*}
  m&=2400\,kg \\
  c&=12000\,\frac{Ns}{m} \\
  k&=15000\,\frac{N}{m} \\
  F_{0}&=17000\,N
\end{align*}

Approximate the acceleration due to gravity as $g=10\,m/s^2$ to
simplify your calculations.

Find the values of all the undetermined coefficients but do not find
the values of the arbitrary constants (do not apply boundary
conditions).

The equation is
$$
2400y''+12000y'+15000y=-24000+17000e^{-2.5t}
$$
The general solution is
$$
c_1e^{-2.5t}+tc_2e^{-2.5t}
$$
The particular solution is
$$
-\frac{8}{5}+\frac{85}{24}t^2e^{-2.5t}
$$

Many students ignore the constant term $-mg$. In 2022, I gave 3.5 points if all else was correct. Too strict?